programs/config_api/src/config_processor.rs

```diff
@@ -138,7 +150,7 @@ mod tests {
 
         let my_config = MyConfig::new(42);
 
-        let instruction = config_instruction::store(&config_pubkey, keys.clone(), &my_config);
+        let instruction = config_instruction::store(&config_pubkey, true, keys.clone(), &my_config);
```

Comment:
no way to add a signer or re-assign signers (only remove them back to the account keypair)?

if this mechanism is supposed to be present, can you describe the procedure?

if the mechanism is deliberately absent, what's the rationale?

Reply:
@rob-solana , I don't understand your question. Are you asking about adding or re-assigning signers from the built Instruction?
Or adding/removing signers when updating the Config account data?